server/Assignments_filler/sheetFiller.py

```diff
@@ -0,0 +1,237 @@
+from __future__ import print_function
```

Comment:
Remove, we are on Python 3, you don't need a print function, you already have it.